Add _posts/2022-08-30-sept-comm-call-on-groups-summary.md (#944)
USRSE/usrse.github.io: _posts/2022-08-30-sept-comm-call-on-groups-summary.md

# Summary of Our September Community Call on Groups in US-RSE

US-RSE is a community driven organization,
but do you know all the groups in the community?
You may have heard someone talking about a “working group”,
but do you know what that means?

This month’s community call was all about groups within US-RSE!
It’s the first in a series we’re calling “Getting to know US-RSE”.

> The call features short talks from all the current working groups,

Reviewer: ```suggestion
The call featured short talks from all the current working groups,
```

as well as a list of skills that Research Software Engineers might possess.
These resources are available on the working group web page.

We are currently focused on establishing a seminar series
consisting of technical talks and tutorials.
These seminars will focus on teaching relevant skills
and best practices to members of the community.
We plan to alternate between technical talks and hands-on tutorials.
Our first technical talk will be "Agile Project Management for Research Software"
by Angel Hering scheduled for Wednesday, October 19 at 4pm ET (1pm PT). We hope to see you there!

If you would like to participate in the the working group,
let us know on Slack in the #wg-education_training
or #education-training channels.
You are also welcome to join the monthly zoom call.

### Diversity, Equity, and Inclusion
#### Co-Chairs: Lance Parsons and TBD
* [Presentation slide](https://docs.google.com/presentation/d/1m5sW0K3H5T_8HQXTJ28io-B8Df2kRP5DAW7xlRF_KkI/edit#slide=id.g15135ed554c_0_1)
* Slack channel (working group): #wg-diversity-equity-inclusion
* Slack channel (general discussion of DEI in RSE): #dei-discussion
* [Page on US-RSE site](https://us-rse.org/wg/dei/)

> * Email: wg-dei@us-rse.org

Reviewer: ```suggestion
* Email: [wg-dei@us-rse.org](mailto:wg-dei@us-rse.org)
```

The US-RSE Diversity, Equity, and Inclusion (DEI) working group is dedicated to
supporting DEI through education, outreach,
and more as described in our mission statement.

The DEI working group believes that continuous education is crucial to

> improving the diversity, equity and inclusion in our community. To that end,

Reviewer: ```suggestion
improving the diversity, equity, and inclusion in our community. To that end,
```

the group ogranizes a regular DEI Speaker Series where we invite experts to
provide a broad perspective on a variety of DEI related issues. A regular DEI
Media Club also provides members an opportunity for a deeper discussion around
relevant books, podcasts, and videos. Those discussions also continue on
the #dei-discussion channel in Slack.

The group is currently working on a couple of initiatives.
The first is building up a list of DEI Resources into a set of Resource Toolkits.
Each toolkit will focus on how DEI issues affect a specific aspect of being an RSE
such as managing and hiring or the various pathways to becoming an RSE.

The other initiative is to collect demographic information
on the US-RSE membership and use it as a benchmark
to see how the organization changes over time.
Knowing some basic information about our membership
will be very helpful in measuring the success of our efforts
and help us plan activities, decide on speaker invitations, resource topics, etc.

Interested members are encouraged to hop on the #dei-discussion channel
and the #wg-diversity-equity-inclusion channel.
We meet on Zoom every two weeks, Wednesdays at 3 PM ET / 12 PM PT.

### Outreach

> #### Miranda Mundt and Josh Teves

Reviewer: ```suggestion
#### Co-Chairs: Miranda Mundt and Josh Teves
```

* [presentation slide](https://docs.google.com/presentation/d/1wRVREocSYy_E_40vXcWQAzSbWXEcwg3r/edit#slide=id.p1)
* Slack channel: #wg-outreach
* Slack group: @wg-or
* [Page on US-RSE site](https://us-rse.org/wg/outreach/)

> * Email address: wg-outreach@us-rse.org

Reviewer: ```suggestion
* Email: [wg-outreach@us-rse.org](mailto:wg-outreach@us-rse.org)
```

The US-RSE Outreach Working Group (officially started in early 2022)
is dedicated to broadening US-RSE’s engagement with the
research software engineer (RSE) and
wider computational science and engineering (CSE) communities.
Its members strive to raise awareness of US-RSE as an association and
RSE as a professional designation through engagement
with industry, academia, national labs, and students.

During the US-RSE September Community Call,
co-chairs Miranda Mundt (Sandia National Laboratories)

> and Joshua Teves (National Institute of Health) detailed the engagement

Reviewer: ```suggestion
and Joshua Teves (National Institutes of Health) detailed the engagement
```

of the Outreach WG so far this year at SciPy (Birds of a Feather Panel),
eScience (Workshop), and Supercomputing (Workshop).

In addition to formal forums, the Outreach WG hosted the first-ever
US-RSE Swag Giveaway event in August 2022.
Of the over 80 submissions received,
28 were randomly selected to receive either a US-RSE tumbler
or a US-RSE backpack. All entrants also received official US-RSE stickers.

Starting soon, the Outreach WG is beginning a new webinar “Funder Series,”
a speaker series aimed at bringing representatives from funders (e.g., NSF, NIH)
 to speak to members of US-RSE. These funders will bring essential information
about opportunities, strategies, and starting points for finding funding.
The announcement for the first talk is here:
<https://us-rse.org/events/2022/2022-10-funder-talk-series/>
More information will be released as this effort evolves.

Finally, the Outreach WG needs your help!
They are calling on members of US-RSE to submit short recorded videos
for use in their “Rapid Interviews” effort.
These recordings are answers to a series of questions
that will give watchers a quick insight into the members of
US-RSE. Instructions can be found

> [here](https://docs.google.com/document/d/1Q_D1mf8h5ryqy4y73j16g2-HY7wp-pBsIQj7O6k9u4s/edit)

Reviewer: ```suggestion
[here](https://docs.google.com/document/d/1Ee21H6Ql9aYzrXQXQ90Y8RHXqqgmxlEsjTPbRLpAxSo/edit?usp=sharing)
```

and the form to submit is found

> [here](https://forms.gle/2niqzDcQUHPAij246).

Reviewer: ```suggestion
[here](https://forms.gle/qcg6AjpLAFxvXy879).
```

### Website
#### Co-Chairs: TBD (Ian Cosden standing in)

> * [Presentation Slide](https://docs.google.com/presentation/d/1K1YJ6pZ0CtiE2kYTiH_4ivQKvUrRZxb63Z6bw7X9gPQedit#slide=id.g15135ed554c_0_1)

Reviewer: ```suggestion
* [Presentation Slide](https://docs.google.com/presentation/d/1K1YJ6pZ0CtiE2kYTiH_4ivQKvUrRZxb63Z6bw7X9gPQ/edit#slide=id.g15135ed554c_0_1)
```

Author: Thank you! :raised_hands:

* Slack channel: #website
* [Page on US-RSE site](https://us-rse.org/wg/website/)

The US-RSE website is the organizations portal to the world.
It’s often the first place people hear of us and sometimes
the first place they hear the term RSE.
It serves as the primary place where we collect and provide resources
and parts of the website (job board, event calendar, etc)
are regularly updated and visited by members of the community.
The website group, not surprisingly, oversees
all facets of the us-rse.org website.
This includes everything from full website design,
reviewing PRs from other groups and contributors,
making functional improvements, and more.
The website is all hosted on github at https://github.com/USRSE/usrse.github.io.
We are looking for more volunteers to help in every aspect
including organizing and chairing the group.
You don’t need to be an expert,
or even familiar with the specific technologies to make an impact!
We can help get you started.

### Elevating RSEs

> #### Kenton McHenry

Reviewer: ```suggestion
#### Chair: Kenton McHenry
```

1. highlighting one story in our newsletter each month,

2. compiling a slide deck (here) with these stories
that we can pull slides from when we talk about the importance of RSEs
with our colleagues, collaborators, and leadership, and

3. collecting outreach channels we should aim to include for
our newsletters and other outreach activities.

Longer term plans include other activities along this front
such as an RSE conference where submitted papers are peer reviewed
on both the science and the RSE aspects of the effort.
If you have other ideas and are interested in being part of this working group,
please join the #wg-elevate-rse channel in the USRSE slack.

## The rest of the call and future efforts

Several other groups that are forming spoke,
such as Financial Stability and the group that organizes community calls.
We also talked briefly about a draft of guidelines for forming groups in US-RSE
that came out of our first community workshop in March.
This included a discussion of the different types of groups
that members can form, how new groups can be formed,
how we’re testing the suggested guidelines,
and how we’d like to promote more affinity groups within US-RSE.

> To find out more about getting involved,

Reviewer: ```suggestion
To find out more about getting involved
```

> and to find out about the next community call on

Reviewer: ```suggestion
and the next community call on
```

It’s the first in a series we’re calling “Getting to know US-RSE”.
The call features short talks from all the current working groups,
including Education & Training; Outreach; and Diversity, Equity, and Inclusion.

In this blog post we want to share with you a little bit
from the call, about five of the working groups in US-RSE.
Our gaol is to give you a sense of where the community is putting its efforts,
and help you understand how you can get involved.

## US-RSE Working Groups

### Education & Training
#### Co-Chairs: Jeff Carver and Lauren Milechin

* [Presentation slide](https://docs.google.com/presentation/d/1Nmp-aPwKB6RH8sBkbeA2IuYiUN5xH24l/edit#slide=id.p1)
* Slack channel (working group): #wg-education_training
* Slack channel (general discussion): #education-training
* [Page on US-RSE site](https://us-rse.org/wg/education_training/)

The goal of the Education and Training working group
is to facilitate collaboration among US-RSE members
who have an a group with an interest in training and education of RSEs.
Our goal is to develop resources and strategies
that support RSE education and training and contribute to the establishment
of a more formal career path for RSEs.

> We meet monthly over Zoom to discuss and work on ongoing efforts.

Reviewer 2: ```suggestion
We meet the first Tuesday of each month at 11:30am ET on Zoom to discuss and work on ongoing efforts.
```

In the past this work has included compiling lists of training links

> as well as a list of skills that a Research Software Engineering might possess.

Reviewer 2: ```suggestion
as well as a list of skills that Research Software Engineers might possess.
```

These resources are available on the working group web page.

We are currently focused on establishing a seminar series

> that will consist of both technical talks and tutorials.

Reviewer 2: ```suggestion
consisting of technical talks and tutorials.
```

These seminars will focus on teaching relevant skills

> and best practices for the community.

Reviewer 2: ```suggestion
and best practices to members of the community.
```

We plan to alternate between technical talks and hands-on tutorials.
Our first technical talk will be "Agile Project Management for Research Software"

> by Angel Hering on Wednesday, October 19 at 4pm ET (1pm PT), we hope to see you there!

Reviewer 2: ```suggestion
by Angel Hering scheduled for Wednesday, October 19 at 4pm ET (1pm PT). We hope to see you there!
```

If you would like to participate in the the working group,
let us know on Slack in the #wg-education_training
or #education-training channels.

> We meet over Zoom the first Tuesday of every month at 11:30am ET,

Reviewer 2: ```suggestion

```

> and we would be happy to have you join.

Reviewer 2: ```suggestion
You are also welcome to join the monthly zoom call.
```

### Diversity, Equity, and Inclusion
#### Co-Chairs: Lance Parsons and TBD
* [Presentation slide](https://docs.google.com/presentation/d/1m5sW0K3H5T_8HQXTJ28io-B8Df2kRP5DAW7xlRF_KkI/edit#slide=id.g15135ed554c_0_1)
* Slack channel (working group): #wg-diversity-equity-inclusion
* Slack channel (general discussion of DEI in RSE): #dei-discussion
* [Page on US-RSE site](https://us-rse.org/wg/dei/)
* Email: wg-dei@us-rse.org

The US-RSE Diversity, Equity, and Inclusion (DEI) working group is dedicated to
supporting DEI through education, outreach,
and more as described in our mission statement.

> As part of the effort to continually educate ourselves through research on
> best practices focused on diversity, equity, and inclusion.
> The group organizes a regular DEI Speaker Series
> providing a diverse and broad perspective on best practices and issues
> focused on diversity, equity, and inclusion.
> The DEI Media Club provides members an opportunity
> for a deeper discussion around relevant books, podcasts, and videos.
> Those discussions can also continue on the #dei-discussion channel.

Author: making this change but fixing a couple typos + tweaking wording just a little

```suggestion
The DEI working group believes that continuous education is crucial to
improving the diversity, equity and inclusion in our community. To that end,
the group ogranizes a regular DEI Speaker Series where we invite experts to
provide a broad perspective on a variety of DEI related issues. A regular DEI
Media Club also provides members an opportunity for a deeper discussion around
relevant books, podcasts, and videos. Those discussions also continue on
the #dei-discussion channel in Slack.
```

The group is currently working on a couple of initiatives.

> The first is building up a list of DEI Resources into a set of Resource Tool-Kits.

Reviewer 3: ```suggestion
The first is building up a list of DEI Resources into a set of Resource Toolkits.
```
